### PR TITLE
Add Google KMS docker files

### PR DESCRIPTION
**Pull request by guykeller**

### CODE REVIEW

**Comment by guykeller** [comment]
Fixed @glicht

**Comment by glicht** [comment]
Please check if you can use the alpine based base docker image

**Comment by roysagi** [comment]
@glicht If I try using alpine for the docker it throws an error saying I'm missing a C++ library (specifically libstdc++.so.6). 

Here is the full error:
Error from Google Key Management Service is : Script failed to run: 
Error: [Traceback (most recent call last):
   
   File "<string>", line 2, in <module>
   File "/usr/local/lib/python3.7/site-packages/google/cloud/kms_v1/__init__.py", line 22, in <module>
    from google.cloud.kms_v1.gapic import key_management_service_client
   File "/usr/local/lib/python3.7/site-packages/google/cloud/kms_v1/gapic/key_management_service_client.py", line 25, in <module>
    import google.api_core.gapic_v1.client_info
   File "/usr/local/lib/python3.7/site-packages/google/api_core/gapic_v1/__init__.py", line 16, in <module>
    from google.api_core.gapic_v1 import config
   File "/usr/local/lib/python3.7/site-packages/google/api_core/gapic_v1/config.py", line 23, in <module>
    import grpc
   File "/usr/local/lib/python3.7/site-packages/grpc/__init__.py", line 23, in <module>
    from grpc._cython import cygrpc as _cygrpc
 ImportError: Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/lib/python3.7/site-packages/grpc/_cython/cygrpc.cpython-37m-x86_64-linux-gnu.so)
] (2604) (2603)

If I use debian it works perfectly.

File: docker/google-kms/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+FROM demisto/python3-deb:3.7.4.1895
```
Comment (glicht) [inline docker/google-kms/Dockerfile]:
```suggestion
FROM demisto/python3-deb:3.7.4.2245
```

**Comment by glicht** [comment]
Looks fine

**Pull request merged**